models/Message.php

```diff
@@ -86,8 +86,8 @@ public function toLocale($locale = null, $message)
         if (!$message) {
             unset($data[$locale]);
         }
-
         $this->message_data = $data;
+
```

Comment:
What was the reason for this whitespace change?

Reply:
Likely just a mistake.